src/main/kotlin/uk/gov/justice/hmpps/architecture/annotations/Tags.kt

```diff
@@ -16,7 +16,12 @@ enum class Tags : addTo {
   },
   PLANNED,
   DEPRECATED,
-  SOFTWARE_AS_A_SERVICE;
+  SOFTWARE_AS_A_SERVICE,
+  DATA_API,
+  DOMAIN_API,
+  AUTH_API,
```

Comment:
Authentication sounds like a capability rather than a kind of API. Perhaps `CAPABILITY_AUTH`?
Maybe that should be a domain annotation similar to `ProblemArea`

Reply:
Yes agree - I've created a capability annotation and use `Identity` for the auth-related services. Might not be exactly right but it's better that the tag